services/cd-service/pkg/repository/repository.go

```diff
@@ -323,7 +325,7 @@ func New(ctx context.Context, cfg RepositoryConfig) (Repository, error) {
 	if err != nil {
 		return nil, err
 	}
-	go bg(ctx, nil)
+	go bg(ctx, nil) //nolint: errcheck
```

Comment:
we should print the error. It might be easier to do that when `bg` does not return an error, but just prints the error itself.
Or created a wrapper that handles the logging of the error.

Reply:
This function is only used in tests, so I just chose to ignore it.